assets/app/features/upgrade/controllers/upgrade-upgrade-administration-server.controller.spec.js

```diff
@@ -175,22 +29,10 @@ describe('Upgrade Flow - Upgrade Administration Server Controller', function ()
     });
 
     describe('on controller creation', function () {
-        it('should set running to true if the upgrade is running', function () {
-            activeStatusResponse.data = incompleteUpgradeResponseData;
-            // recreate the controller so it can pick our modified initialResponse
-            controller = $controller('UpgradeUpgradeAdministrationServerController');
-            $rootScope.$digest();
-            // initial model should have changed based on the initialization response
-            assert.isFalse(controller.adminUpgrade.completed);
-            assert.isTrue(controller.adminUpgrade.running);
-        });
-
-        it('should set completed to true if upgrade is completed', function () {
-            activeStatusResponse.data = completedUpgradeResponseData;
-            controller = $controller('UpgradeUpgradeAdministrationServerController');
-            $rootScope.$digest();
-            assert.isTrue(controller.adminUpgrade.completed);
-            assert.isFalse(controller.adminUpgrade.running);
+        it('should call syncStatusFlags() to update the state', function () {
+            expect(upgradeStatusFactory.syncStatusFlags).toHaveBeenCalledWith(
+                'admin_upgrade', controller.adminUpgrade, jasmine.any(Function), jasmine.any(Function)
```

Comment:
`waitForUpgradeToEnd` is not used outside of the controller. I don't like exposing things just for tests.
The second part is done.